app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -52,11 +54,11 @@ class SyncIndicatorViewController @Inject constructor(
     val syncProgress = lastSyncedStateStream.filter { it.lastSyncProgress != null }
 
     val showSyncIndicatorState = Observables
-        .combineLatest(screenCreated, syncProgress)
-        { _, stateStream ->
+        .combineLatest(screenCreated, syncProgress, configProvider.toObservable())
```

Comment:
Vinay has started using `Observable` instead of `Single` for configs because we anyway convert them to observables using `configProvider.toObservable()`. Do you want to do the same thing?

Reply:
Done